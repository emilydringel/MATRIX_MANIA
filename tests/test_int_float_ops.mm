/* operators on ints and floats */ 

def void testintflot(int a, float b){
    print(a + b);
    print(a - b);
    print(a * b);
    print(a / b);
    print(a == b);
    print(a == a);
    print(a != b);
    print(a != a);
    print(a > b);
    print(a >= b);
    print(a < b);
    print(a <= b);
}
def int main(){
    
    int c = 80;
    float d = 5.5;

    testintfloat(c, d);

    testintfloat(d, d);
}
