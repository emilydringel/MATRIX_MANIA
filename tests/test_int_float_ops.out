85.5
74.5
440
14.5454545455
0
1
0
1
1
1
0
0
